src/AspNet.Security.OAuth.Discord/DiscordAuthenticationConstants.cs

```diff
@@ -23,5 +23,10 @@ public static class Claims
             public const string AvatarUrl = "urn:discord:avatar:url";
             public const string Discriminator = "urn:discord:user:discriminator";
         }
+
+        public static class Key
```

Comment:
Should that class be called `Parameters`? Alternatively, we can also remove it and hardcode the "prompt" parameter in the handler, as I'm not sure anyone will use this constant.

Reply:
Okay, sure.  I'll just remove it.